turbinia/client.py

```diff
@@ -1120,8 +1122,12 @@ def __init__(self, jobs_denylist=None, jobs_allowlist=None):
 
   def start(self):
     """Start Turbinia Server."""
-    log.info('Starting Prometheus endpoint.')
-    start_http_server(port=config.PROMETHEUS_PORT, addr=config.PROMETHEUS_ADDR)
+    if config.PROMETHEUS_PORT and config.PROMETHEUS_ADDR:
```

Comment:
I'll add a PROMETHEUS_ENABLE config setting in another PR later.

Reply:
Sounds good, thanks!